CONTRIBUTING.md

```diff
@@ -1 +1,157 @@
-Please refer to the [Gardener contributor guide](https://gardener.cloud/docs/contribute/).
+---
+title: Contribute
+description: Contributors guides for code and documentation
```

Comment:
This is org wide document and it should not be moved here. 
Probably https://github.com/gardener/community is better place ...

Reply:
I see 2 options:
- every component has its own contribute guide (the approach proposed by Kristian)
- we have one common org wide document (in which case we shall remove the component specific contribution files or at least replace the common part with a link to the central one and leave only component specifics)

I also started with the assumption that one common org wide guide will be preferable, but I've heard other opinions as well.